verification/src/http_server/routes.rs

```diff
@@ -0,0 +1,20 @@
+use actix_web::web;
+
+use crate::http_server::handlers::{
+    health::get_health,
+    verifiers::{source_code, sourcify, standard_json, viper},
+};
+
+pub fn routes(service_config: &mut web::ServiceConfig) {
+    service_config
+        .route("/health", web::get().to(get_health))
+        .service(
+            web::scope("/api/v1").service(
+                web::scope("verification")
+                    .route("/flatten/", web::post().to(source_code::verificate))
+                    .route("/sourcify/", web::post().to(sourcify::verificate))
+                    .route("/viper/", web::post().to(viper::verificate))
+                    .route("/standard_json/", web::post().to(standard_json::verificate)),
+            ),
```

Comment:
1. It seems that we make them to be GET requests. I believe it would be better, as read-only nature of our service does not appear via POST requests. Also, GET requests are usually easier to handle and process (e.g. if we want to add caching, POST requests may be not a suitable option).
2. `../verification/viper/` and `../verification/viper` represents different resources. Trailing `/` creates a new empty path at the end, which is not what you actually want, I believe.